requirements.txt

```diff
@@ -1,4 +1,5 @@
 numpy
```

Comment:
You can revert the changes in `requirements.txt` and `setup.py` as they are not used in the main branch.

Reply:
Do you mean that I should revert them to the last commit before I opened this branch?

Reply:
Resolved in my commits